src/components/ProductList.js

```diff
@@ -3,8 +3,34 @@ import CalendarButton from "./CalendarButton";
 import ProductCard from "./ProductCard";
 import CategoryFilter from "./CategoryFilter";
 import "./ProductList.css";
+import { useEffect, useState } from "react";
 
 export default function ProductList() {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    const fetchProducts = () => {
+      return fetch("products.json")
+        .then((response) => response.json())
+        .then((productData) => {
+          setProducts(productData);
+          console.log(products);
```

Comment:
This should be removed before merging into main 😉

Reply:
Hi Anneke, thanks for your review. Good catch with the console.log, it actually solved my useEffect issue :)